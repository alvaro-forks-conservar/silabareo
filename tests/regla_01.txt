casa
hace
casi
caso
cáñamo
pláceme
parásito
ágora
pábulo
tierra
cierre
decisión
eso
acéfalo
cadavérico
pésol
émula
cita
guise
piso
ilusión
jenízaro
íride
foca
moreno
zoco
ocupar
esófago
jóvenes
pócima
podólogo
óvulo
ocupar
sucio
uso
húmedo
lúcido
parahúso
úvula
asunto
penumbra
aéreo
ignífero
ignívomo
ignífugo
morisco
auyentar
usufructo
inmiscúyase