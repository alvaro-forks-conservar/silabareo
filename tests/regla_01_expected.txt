ca-sa
ha-ce
ca-si
ca-so
cá-ña-mo
plá-ce-me
pa-rá-si-to
á-go-ra
pá-bu-lo
tie-rra
cie-rre
de-ci-sión
e-so
a-cé-fa-lo
ca-da-vé-ri-co
pé-sol
é-mu-la
ci-ta
gui-se
pi-so
i-lu-sión
je-ní-za-ro
í-ri-de
fo-ca
mo-re-no
zo-co
o-cu-par
e-só-fa-go
jó-ve-nes
pó-ci-ma
po-dó-lo-go
ó-vu-lo
o-cu-par
su-cio
u-so
hú-me-do
lú-ci-do
pa-ra-hú-so
ú-vu-la
a-sun-to
pe-num-bra
a-é-re-o
ig-ní-fe-ro
ig-ní-vo-mo
ig-ní-fu-go
mo-ris-co
au-yen-tar
u-su-fruc-to
in-mis-cú-ya-se